internal/controller/apiextensions/composite/reconciler_test.go

```diff
@@ -42,6 +45,7 @@ import (
 
 func TestReconcile(t *testing.T) {
 	errBoom := errors.New("boom")
+	testLog := logging.NewLogrLogger(zap.New(zap.UseDevMode(true), zap.WriteTo(io.Discard)).WithName("testlog"))
```

Comment:
Could be handy to define this in the logging package in future.

Reply:
@negz agree 👍🏻 was planning on opening an issue on `crossplane-runtime`

Reply:
I forgot about this PR and was wondering why this pattern was everywhere. Definitely still feel like this would be good to have in crossplane-runtime. In fact we could probably make the NopLogger that everything uses by default use this pattern so that we'd detect panics without having to explicitly plumb it in everywhere.